src/utils/load-commands.js

```diff
@@ -68,7 +68,6 @@ function requireCommands () {
   cmds.util = (send, config) => {
     return {
       getEndpointConfig: require('../util/get-endpoint-config')(config),
-      crypto: require('libp2p-crypto'),
```

Comment:
README needs to be updated with this change and we need appropriate "BREAKING CHANGE" sections in the commits for this (as well as the switch to bignumber.js) so that they are reflected in the changelog when we release.

Reply:
@hugomrdias we need to do this please ^^